pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -185,7 +191,7 @@ type Build struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   BuildSpec   `json:"spec,omitempty"`
+	Spec   BuildSpec   `json:"spec"`
```

Comment:
Note that technically this is a breaking API change - this needs to be in the release note.